### PR TITLE
Add a getter for Toxiproxy exposed port

### PR DESCRIPTION
**Pull request by lutovich**
Hello!
I used Toxiproxy container to simulate a connection cut between two different Docker containers. It was a bit hard to determine what exposed port of the proxy corresponds to the exposed port of the target container. The code I ended up with did `toxiproxy.getExposedPorts().get(1)` which is not very intuitive because `toxiproxy.getExposedPorts().get(0)` is the proxy control port. Thus this PR. Hope it makes sense :)

Before this change, there only existed a getter for the mapped port which is useful for accessing the container from the host. A new getter can be useful when a Toxiproxy is setup between two different Docker containers to simulate network instabilities.

### CODE REVIEW

**Comment by rnorth** [comment]
This is a good idea, but I just want to have a think about naming - I think that `getExposedProxyPort()` could be improved upon. It definitely deserves some inline Javadocs and a general documentation update to explain what this is for and when to use it 😄

**Comment by lutovich** [comment]
Hi @rnorth, I agree, the name could be improved. What do you think about `getDockerProxyPort` or `getContainerProxyPort`?

**Comment by bsideup** [comment]
@rnorth soft ping :)

**Comment by rnorth** [comment]
Sorry - naming this is hard! I think personally I'd prefer `getOriginalProxyPort` or `getDirectProxyPort`. We should also have really solid documentation for when you should and shouldn't use this, both in the form of Javadocs and in the published markdown docs.

I think we should probably skip this for today's release - sorry that my delay in  getting back slowed things down.

**Comment by lutovich** [comment]
Thanks, `getOriginalProxyPort` sounds good to me. I'll change the PR and add docs this week.

**Comment by lutovich** [comment]
Hi @rnorth, this PR is now updated:
 * changed getter name from `getExposedProxyPort` to `getOriginalProxyPort`
 * added Javadocs and markdown docs for Toxyproxy port-related methods
 * attempted to fix flaky Toxiproxy tests by increasing the default Jedis timeout

Please re-review, thanks in advance :)

File: docs/modules/toxiproxy.md [hunk 1]
```diff
@@ -29,13 +29,21 @@ We do this as follows:
 [Starting proxying connections to a target container](../../modules/toxiproxy/src/test/java/org/testcontainers/containers/ToxiproxyTest.java) inside_block:obtainProxyObject
 <!--/codeinclude-->
 
-Then, to establish a connection via Toxiproxy, we obtain **Toxiproxy's** proxy host IP and port:
+To establish a connection from the test code (on the host machine) to the target container via Toxiproxy, we obtain **Toxiproxy's** proxy host IP and port:
 
 <!--codeinclude-->
-[Obtaining proxied host and port](../../modules/toxiproxy/src/test/java/org/testcontainers/containers/ToxiproxyTest.java) inside_block:obtainProxiedHostAndPort
+[Obtaining proxied host and port for the host mahcine](../../modules/toxiproxy/src/test/java/org/testcontainers/containers/ToxiproxyTest.java) inside_block:obtainProxiedHostAndPortForHostMachine
```
Comment (rnorth) [inline docs/modules/toxiproxy.md:35]:
```suggestion
[Obtaining proxied host and port for connections from the host machine](../../modules/toxiproxy/src/test/java/org/testcontainers/containers/ToxiproxyTest.java) inside_block:obtainProxiedHostAndPortForHostMachine
```

File: docs/modules/toxiproxy.md [hunk 2]
```diff
@@ -29,13 +29,21 @@ We do this as follows:
...
 <!--/codeinclude-->
 
-Code under test, or other containers, should connect to this proxied host IP and port.
+Code under test should connect to this proxied host IP and port.
+
+To establish a connection from a different container on the same network to the target container via Toxiproxy, we use **Toxiproxy's** network alias and original port:
+
+<!--codeinclude-->
+[Obtaining proxied host and port for a different container](../../modules/toxiproxy/src/test/java/org/testcontainers/containers/ToxiproxyTest.java) inside_block:obtainProxiedHostAndPortForDifferentContainer
```
Comment (rnorth) [inline docs/modules/toxiproxy.md:43]:
```suggestion
[Obtaining proxied host and port for connections from a different container](../../modules/toxiproxy/src/test/java/org/testcontainers/containers/ToxiproxyTest.java) inside_block:obtainProxiedHostAndPortForDifferentContainer
```

File: modules/toxiproxy/src/main/java/org/testcontainers/containers/ToxiproxyContainer.java [hunk 3]
```diff
@@ -99,8 +100,21 @@ public ContainerProxy getProxy(String hostname, int port) {
         private static final String CUT_CONNECTION_DOWNSTREAM = "CUT_CONNECTION_DOWNSTREAM";
         private static final String CUT_CONNECTION_UPSTREAM = "CUT_CONNECTION_UPSTREAM";
         private final Proxy toxi;
+        /**
+         * The IP address that this proxy container may be reached on.
```
Comment (rnorth) [inline modules/toxiproxy/src/main/java/org/testcontainers/containers/ToxiproxyContainer.java:104]:
```suggestion
         * The IP address that this proxy container may be reached on from the host machine.
```

**Comment by lutovich** [comment]
@rnorth thanks a lot! I committed all suggestions and squashed them into one commit

**Pull request merged**